plugins/Signatures/class.signatures.plugin.php

```diff
@@ -255,16 +255,15 @@ public function checkSignatureLength($Values, &$Sender) {
      */
     public function checkNumberOfImages($Values, &$Sender) {
         $maxImages = self::getMaximumNumberOfImages();
+        $max = c('Plugins.Signatures.MaxNumberImages');
```

Comment:
1) Is there a config still named this? I thought Stephane just changed the namespace for all these last month. 2) `$max` isn't enough context for a variable name when 'max' is used 4 times in 2 lines.